satellite/console/consoleweb/consoleql/mail.go

```diff
@@ -8,6 +8,8 @@ const (
 	ActivationPath = "activationPath"
 	// PasswordRecoveryPath is key for path which handles password recovery
 	PasswordRecoveryPath = "passwordRecoveryPath"
+	// PasswordRecoveredAccidentallyPath is key for path which handles let us know sequence
+	PasswordRecoveredAccidentallyPath = "passwordRecoveryPathLetUsKnowPath"
```

Comment:
This isn't really clear on what it is.

Reply:
Maybe `CancelPasswordRecoveryPath` or `PasswordRecoveryCancelPath`?

Reply:
First option sounds best on my opinion. Thanks for suggestion.